[2020-08-22T14:11:39.932] [ERROR] custom - Error: asdasddfjshdflkashflkahdsf
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:8:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at Application.handleRequest (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa/lib/application.js:168:12)
    at Server.handleRequest (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa/lib/application.js:150:19)
    at Server.emit (events.js:310:20)
    at parserOnIncoming (_http_server.js:784:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:119:17)
[2020-08-22T14:11:59.374] [ERROR] custom - Error: asdasddfjshdflkashflkahdsf
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:8:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at Application.handleRequest (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa/lib/application.js:168:12)
    at Server.handleRequest (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa/lib/application.js:150:19)
    at Server.emit (events.js:310:20)
    at parserOnIncoming (_http_server.js:784:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:119:17)
[2020-08-22T15:31:25.105] [ERROR] custom - ReferenceError: data is not defined
    at ErrorController.saveErrorUsePost (/Users/dilomen/Documents/my_github_project/frontbug/node/controller/error.js:31:63)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/router/index.js:12:70
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:368:16
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:373:31)
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:24:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa2-useragent/dist/index.js:37:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async bodyParser (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:14:3
[2020-08-22T15:34:11.421] [ERROR] custom - ReferenceError: data is not defined
    at ErrorController.saveErrorUsePost (/Users/dilomen/Documents/my_github_project/frontbug/node/controller/error.js:31:57)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/router/index.js:12:70
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:368:16
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:373:31)
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:24:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa2-useragent/dist/index.js:37:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async bodyParser (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:14:3
[2020-08-22T15:34:44.775] [ERROR] custom - ReferenceError: data is not defined
    at ErrorController.saveErrorUsePost (/Users/dilomen/Documents/my_github_project/frontbug/node/controller/error.js:31:57)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/router/index.js:12:70
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:368:16
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:373:31)
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:24:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa2-useragent/dist/index.js:37:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async bodyParser (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:14:3
[2020-08-22T15:38:07.884] [ERROR] custom - ReferenceError: data is not defined
    at ErrorController.saveErrorUsePost (/Users/dilomen/Documents/my_github_project/frontbug/node/controller/error.js:31:57)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/router/index.js:12:70
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:368:16
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:373:31)
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:24:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa2-useragent/dist/index.js:37:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async bodyParser (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:14:3
[2020-08-22T15:39:11.507] [ERROR] custom - ReferenceError: data is not defined
    at ErrorController.saveErrorUsePost (/Users/dilomen/Documents/my_github_project/frontbug/node/controller/error.js:29:14)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/router/index.js:12:70
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:368:16
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:373:31)
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:24:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa2-useragent/dist/index.js:37:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async bodyParser (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:14:3
[2020-08-22T15:39:33.801] [ERROR] custom - ReferenceError: result is not defined
    at ErrorController.saveErrorUsePost (/Users/dilomen/Documents/my_github_project/frontbug/node/controller/error.js:31:16)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/router/index.js:12:70
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:368:16
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:373:31)
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:24:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa2-useragent/dist/index.js:37:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async bodyParser (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:14:3
[2020-08-22T16:11:59.504] [ERROR] custom - SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at ErrorController.saveErrorUseGet (/Users/dilomen/Documents/my_github_project/frontbug/node/controller/error.js:21:36)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/router/index.js:11:69
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:368:16
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:373:31)
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:24:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa2-useragent/dist/index.js:37:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async bodyParser (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:14:3
[2020-08-22T16:13:35.236] [ERROR] custom - ReferenceError: result is not defined
    at ErrorController.saveErrorUseGet (/Users/dilomen/Documents/my_github_project/frontbug/node/controller/error.js:22:16)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/router/index.js:11:69
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:368:16
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:34:12
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-router/lib/router.js:373:31)
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:24:9
    at dispatch (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-compose/index.js:42:32)
    at /Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa2-useragent/dist/index.js:37:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async bodyParser (/Users/dilomen/Documents/my_github_project/frontbug/node/node_modules/koa-bodyparser/index.js:95:5)
    at async /Users/dilomen/Documents/my_github_project/frontbug/node/app.js:14:3
